benchmarks/Neo.VM.Benchmarks/InstructionBuilder/Instruction.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// Instruction.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Microsoft.CodeAnalysis;
+using Neo.Extensions;
+using Neo.VM.Types;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Numerics;
+using System.Reflection;
+using System.Text;
+
+namespace Neo.VM.Benchmark;
+
+[DebuggerDisplay("{_opCode}")]
+public class Instruction
```

Comment:
Why we need to replicate this class?

Reply:
this is from the compiler, it is different from what VM has, totally different.

Reply:
can we reference the other?

Reply:
> Can we use the current one?

we can not, what i want here is to constrcut the script, which is exactly what compiler instruction does. VM instruction is for execution purpose, they are totally different.